en/asgardeo/docs/get-started/try-samples/ws-federation-webapp.md

```diff
@@ -0,0 +1,137 @@
+# Sample WS-Federation web app
+
+By following this guide, you will be able to deploy a WS-Federation-based web application and enable login for it using the Passive Security Token Service (Passive STS) of {{product_name}}.
+
+!!! info
+    {{product_name}} uses its passive security token service (Passive STS) as its WS-Federation implementation.
+    Passive STS is capable of issuing SAML 1.1 and 2.0 security tokens. To request a SAML 2.0 security token, the Request Security Token (RST) should be sent to the passive STS endpoint with the token type, `SAMLV2.0`. If no RST is specified, {{product_name}} issued a SAML 1.1 token by default.
```

Comment:
```suggestion
    Passive STS is capable of issuing SAML 1.1 and 2.0 security tokens. To request a SAML 2.0 security token, the Request Security Token (RST) should be sent to the passive STS endpoint with the token type, `SAMLV2.0`. If no RST is specified, {{product_name}} issues a SAML 1.1 token by default.
```

Reply:
All the content in this PR is copied directly from the IS docs. @himeshsiriwardana, do we need to update the content?

Reply:
If we have a corresponding IS doc with the same content, let's move that to /includes directory and reuse it for asgardeo using the {% include ""} directive.

Reply:
done